pages/markdown.tsx

```diff
@@ -0,0 +1,55 @@
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import Typography from '@mui/material/Typography';
+import { marked } from 'marked';
+import React, { useState } from 'react';
+
+import Heading from '../components/Heading';
+import Layout from '../components/Layout';
+
+export default function MarkDownPreview() {
+  const [output, setOutput] = useState('');
+
+  return (
+    <Layout title='MarkDownPreview'>
+      <Heading>Markdown Previewer</Heading>
+      <Typography
+        paragraph
+        textAlign='center'
+      >
+        Paste or type markdown style to preview it.
+      </Typography>
+      <Box
+        className='mainBox'
+        display='flex'
+        justifyContent='stretch'
+        gap={3}
+        width='80%'
+        sx={{
+          flexDirection: { md: 'row', xs: 'column' },
+        }}
+      >
+        {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+        <label htmlFor='inputText' />
+        <TextField
+          id='inputText'
```

Comment:
Add a `label` attribute, like the examples here - https://mui.com/material-ui/react-text-field/

Reply:
Let's make that a meaningful label, like "Markdown"?

Reply:
That's why I asked **what** to write... but ok.